src/js/components/Tag/__tests__/Tag-test.tsx

```diff
@@ -235,4 +238,29 @@ describe('Tag', () => {
       'Remove Fruits',
     );
   });
+
+  const customHoverTheme: ThemeType = deepMerge(grommet, {
+    tag: {
+      border: true,
+      hover: {
+        background: 'pink',
+        border: 'green',
+      },
+    },
+  });
+
+  test('theme border and hover background/border', async () => {
+    const user = userEvent.setup();
+
+    const { container } = render(
+      <Grommet theme={customHoverTheme}>
+        <Tag value="Hover me" onClick={() => {}} />
```

Comment:
I think we need to add a border prop to see the border color change come through. Same thing with background.

<img width="447" height="130" alt="Screenshot 2025-09-10 at 12 23 07 PM" src="https://github.com/user-attachments/assets/3c9ea236-c415-4480-9acf-b0f56806b76a" />